### DISCUSSION TITLE
Updating obstacle position with dtTileCache

### DISCUSSION THREAD

**Discussion by Marqin** [recastnavigation/recastnavigation#652] [Q&A]
Hey, so I have lots of externaly controlled entities and was looking into dtTileCache obstacles and found out there is only REQUEST_ADD and REQUEST_REMOVE, but there is no request to update touched tiles if obstacle moves. Does it mean I have to delete and readd my moving entities every detour tick?

**Comment by Marqin** [comment]
Or is there some better alternative for externaly controlled entities in Detour?

**Comment by quetzalsly** [comment]
Hey did you ever figure this out?

**Reply by Marqin** [reply]
not really
